src/Orleans.Streaming.Redis/Storage/RedisDataManager.cs

```diff
@@ -158,61 +199,52 @@ private void CheckAlertSlowAccess(DateTimeOffset startOperation, [CallerMemberNa
         private void OnChannelReceivedData(RedisChannel channel, RedisValue value)
         {
             _queue.Enqueue(value);
+            _logger.Debug("{Count}", _queue.Count);
+
+            // Dequeue until we're below our queue cache limit
+            while (_queue.Count > _options.QueueCacheSize && _queue.TryDequeue(out var _)) { }
         }
 
         private static string SanitizeQueueName(string queueName)
         {
-            var tmp = queueName;
-            tmp = tmp.ToLowerInvariant();
-            tmp = tmp
-                .Replace('/', '-') // Forward slash
-                .Replace('\\', '-') // Backslash
-                .Replace('#', '-') // Pound sign
-                .Replace('?', '-') // Question mark
-                .Replace('&', '-')
-                .Replace('+', '-')
-                .Replace(':', '-')
-                .Replace('.', '-')
-                .Replace('%', '-');
-            return tmp;
+            // Nothing for now
```

Comment:
Is nothing needed (aka Redis is pretty accommodating of names?) or should we do something? The comment's "for now" has me wondering.